diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/arr_script netperf-2.6.0/doc/examples/arr_script
--- netperf-2.6.0.org/doc/examples/arr_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/arr_script	2026-03-03 00:29:48.651211837 +0100
@@ -1,3 +1,4 @@
+#!/bin/sh
 # this is retained mostly for historical purposes and may not be
 # up to date for contemporary netperf versions
 #
@@ -12,8 +13,7 @@ init_parms() {
   TOLERANCE=15
   MAX_RETRIES=3
 
-  NETPERF="/usr/local/netperf/netperf"
-  NETPERF="./netperf"
+  NETPERF="/usr/bin/netperf"
 
   NPROC_LIST=""
   RR_SIZES=""
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/packet_byte_script netperf-2.6.0/doc/examples/packet_byte_script
--- netperf-2.6.0.org/doc/examples/packet_byte_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/packet_byte_script	2026-03-03 00:29:48.651409176 +0100
@@ -25,7 +25,7 @@ if [ $# -eq 0 ]; then
 fi
 
 # where is netperf
-NETPERF_DIR=${NETPERF_DIR:=/opt/netperf2/bin}
+NETPERF_DIR=${NETPERF_DIR:=/usr/bin}
 
 
 # at what port will netserver be waiting? If you decide to run
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/sctp_stream_script netperf-2.6.0/doc/examples/sctp_stream_script
--- netperf-2.6.0.org/doc/examples/sctp_stream_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/sctp_stream_script	2026-03-03 00:29:48.652033668 +0100
@@ -24,7 +24,7 @@ fi
 # where the programs are
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME=.
+NETHOME=/usr/bin
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/snapshot_script netperf-2.6.0/doc/examples/snapshot_script
--- netperf-2.6.0.org/doc/examples/snapshot_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/snapshot_script	2026-03-03 00:29:48.651506380 +0100
@@ -37,7 +37,7 @@
 #
 # where is netperf installed, there are a few possible places:
 
-NETPERF_CMD=${NETPERF_CMD:=/opt/netperf/netperf}
+NETPERF_CMD=${NETPERF_CMD:=/usr/bin/netperf}
 
 
 # there should be no more than two parms passed
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/tcp_range_script netperf-2.6.0/doc/examples/tcp_range_script
--- netperf-2.6.0.org/doc/examples/tcp_range_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/tcp_range_script	2026-03-03 00:29:48.651591809 +0100
@@ -42,9 +42,7 @@ PORT=""
 
 # where is netperf, and are there any "constant" options such as 
 # the netserver port number
-#NETHOME=/usr/etc/net_perf
-NETHOME="."
-NETPERF=$NETHOME/netperf $PORT
+NETPERF="/usr/bin/netperf $PORT"
 
 # How accurate we want the estimate of performance: 
 #      maximum and minimum test iterations (-i)
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/tcp_rr_script netperf-2.6.0/doc/examples/tcp_rr_script
--- netperf-2.6.0.org/doc/examples/tcp_rr_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/tcp_rr_script	2026-03-03 00:29:48.651681802 +0100
@@ -25,7 +25,7 @@ fi
 # where the programs are
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME=.
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/tcp_stream_script netperf-2.6.0/doc/examples/tcp_stream_script
--- netperf-2.6.0.org/doc/examples/tcp_stream_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/tcp_stream_script	2026-03-03 00:29:48.651758775 +0100
@@ -24,7 +24,7 @@ fi
 # where the programs are
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME=.
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a different port than the default of 12865, then set
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/udp_rr_script netperf-2.6.0/doc/examples/udp_rr_script
--- netperf-2.6.0.org/doc/examples/udp_rr_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/udp_rr_script	2026-03-03 00:29:48.651875649 +0100
@@ -22,7 +22,7 @@ fi
 
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME="."
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
diff -urNp -x '*.orig' netperf-2.6.0.org/doc/examples/udp_stream_script netperf-2.6.0/doc/examples/udp_stream_script
--- netperf-2.6.0.org/doc/examples/udp_stream_script	2011-02-10 18:04:20.000000000 +0100
+++ netperf-2.6.0/doc/examples/udp_stream_script	2026-03-03 00:29:48.651952075 +0100
@@ -19,7 +19,7 @@ fi
 
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME="."
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
