--- netperf-2.4.5-scripts/doc/examples/arr_script	2011-02-21 14:04:33.210661848 +0200
+++ netperf-2.4.5-scripts/doc/examples/arr_script	2011-02-21 14:04:33.210661848 +0200
@@ -1,3 +1,4 @@
+#!/bin/sh
 # this is retained mostly for historical purposes and may not be
 # up to date for contemporary netperf versions
 #
@@ -13,6 +14,5 @@
   MAX_RETRIES=3
 
-  NETPERF="/usr/local/netperf/netperf"
-  NETPERF="./netperf"
+  NETPERF="/usr/bin/netperf"
 
   NPROC_LIST=""
--- netperf-2.4.5/doc/examples/packet_byte_script	2007-11-09 01:29:45.000000000 +0200
+++ netperf-2.4.5-scripts/doc/examples/packet_byte_script	2011-02-21 13:32:49.530443373 +0200
@@ -25,7 +25,7 @@
 fi
 
 # where is netperf
-NETPERF_DIR=${NETPERF_DIR:=/opt/netperf2/bin}
+NETPERF_DIR=${NETPERF_DIR:=/usr/bin}
 
 
 # at what port will netserver be waiting? If you decide to run
--- netperf-2.4.5/doc/examples/snapshot_script	2007-06-02 01:06:03.000000000 +0300
+++ netperf-2.4.5-scripts/doc/examples/snapshot_script	2011-02-21 13:32:49.533776462 +0200
@@ -37,7 +37,7 @@
 #
 # where is netperf installed, there are a few possible places:
 
-NETPERF_CMD=${NETPERF_CMD:=/opt/netperf/netperf}
+NETPERF_CMD=${NETPERF_CMD:=/usr/bin/netperf}
 
 
 # there should be no more than two parms passed
--- netperf-2.4.5-scripts/doc/examples/tcp_range_script	2011-02-21 14:05:38.182552753 +0200
+++ netperf-2.4.5-scripts/doc/examples/tcp_range_script	2011-02-21 14:05:38.182552753 +0200
@@ -42,9 +42,7 @@
 
 # where is netperf, and are there any "constant" options such as 
 # the netserver port number
-#NETHOME=/usr/etc/net_perf
-NETHOME="."
-NETPERF=$NETHOME/netperf $PORT
+NETPERF="/usr/bin/netperf $PORT"
 
 # How accurate we want the estimate of performance: 
 #      maximum and minimum test iterations (-i)
--- netperf-2.4.5/doc/examples/tcp_rr_script	2007-06-02 01:06:03.000000000 +0300
+++ netperf-2.4.5-scripts/doc/examples/tcp_rr_script	2011-02-21 13:32:49.533776462 +0200
@@ -25,7 +25,7 @@
 # where the programs are
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME=.
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
--- netperf-2.4.5/doc/examples/tcp_stream_script	2007-06-02 01:06:03.000000000 +0300
+++ netperf-2.4.5-scripts/doc/examples/tcp_stream_script	2011-02-21 13:32:49.533776462 +0200
@@ -24,7 +24,7 @@
 # where the programs are
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME=.
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
--- netperf-2.4.5/doc/examples/udp_rr_script	2007-06-02 01:06:03.000000000 +0300
+++ netperf-2.4.5-scripts/doc/examples/udp_rr_script	2011-02-21 13:32:49.537109551 +0200
@@ -22,7 +22,7 @@
 
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME="."
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
--- netperf-2.4.5/doc/examples/udp_stream_script	2007-06-02 01:06:03.000000000 +0300
+++ netperf-2.4.5-scripts/doc/examples/udp_stream_script	2011-02-21 13:32:49.537109551 +0200
@@ -19,7 +19,7 @@
 
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME="."
+NETHOME="/usr/bin"
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
--- netperf-2.4.5//doc/examples/sctp_stream_script	2007-06-02 01:06:03.000000000 +0300
+++ netperf-2.4.5-scripts/doc/examples/sctp_stream_script	2011-02-21 14:04:53.472505842 +0200
@@ -24,7 +24,7 @@
 # where the programs are
 #NETHOME=/usr/local/netperf
 #NETHOME="/opt/netperf"
-NETHOME=.
+NETHOME=/usr/bin
 
 # at what port will netserver be waiting? If you decide to run
 # netserver at a differnet port than the default of 12865, then set
