--- netperf-2.3.orig/makefile	2004-06-26 00:09:50.000000000 +0200
+++ netperf-2.3/makefile	2004-10-19 10:20:37.153146536 +0200
@@ -16,7 +16,7 @@
 # in /usr/local/netperf or /opt/netperf
 #
 #NETPERF_HOME = /usr/local/netperf
-NETPERF_HOME = /opt/netperf
+NETPERF_HOME = /usr/bin/netperf
 
 # The compiler on your system might be somewhere else, and/or have
 # a different name.
@@ -111,7 +111,7 @@
 #                   the new HUNDRED_USEC
 
 LOG_FILE=DEBUG_LOG_FILE="\"/tmp/netperf.debug\""
-CFLAGS = -O -D$(LOG_FILE) -DNEED_MAKEFILE_EDIT
+CFLAGS = -O -D$(LOG_FILE)
 
 # Some platforms, and some options, require additional libraries.
 # you can add to the "LIBS =" line to accomplish this. if you find
