diff -Nur netperf-2.3.orig/makefile netperf-2.3.makefile/makefile
--- netperf-2.3.orig/makefile	2004-06-26 00:09:50.000000000 +0200
+++ netperf-2.3.makefile/makefile	2006-08-10 13:50:14.000000000 +0200
@@ -16,7 +16,7 @@
 # in /usr/local/netperf or /opt/netperf
 #
 #NETPERF_HOME = /usr/local/netperf
-NETPERF_HOME = /opt/netperf
+NETPERF_HOME = /usr/bin/netperf
 
 # The compiler on your system might be somewhere else, and/or have
 # a different name.
@@ -111,7 +111,7 @@
 #                   the new HUNDRED_USEC
 
 LOG_FILE=DEBUG_LOG_FILE="\"/tmp/netperf.debug\""
-CFLAGS = -O -D$(LOG_FILE) -DNEED_MAKEFILE_EDIT
+CFLAGS = $(OPTFLAGS) -D$(LOG_FILE)
 
 # Some platforms, and some options, require additional libraries.
 # you can add to the "LIBS =" line to accomplish this. if you find
@@ -188,10 +188,10 @@
 all: netperf netserver
 
 netperf:	$(NETPERF_OBJS)
-		$(CC) -o $@ $(NETPERF_OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $@ $(NETPERF_OBJS) $(LIBS)
 
 netserver:	$(NETSERVER_OBJS)
-		$(CC) -o $@ $(NETSERVER_OBJS) $(LIBS)
+		$(CC) $(LDFLAGS) -o $@ $(NETSERVER_OBJS) $(LIBS)
 
 netperf.o:	netperf.c netsh.h makefile
 
